schema.sql

```diff
@@ -76,3 +76,7 @@ CREATE TABLE visits (
   vet_id integer REFERENCES vets(id),
   visit_date date
 );
+
+-- database performance audit -------------------------------------
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```

Comment:
- [ ] Also you have missed a requirement the **```CREATE INDEX ON visits (animal_id); also => CREATE INDEX ON visits (vet_id); also => CREATE INDEX ON owners (email);```**

Reply:
I  am not seeing this missed requirement in the document

Reply:
Hello as I mentioned the index create is not added to your schema

Reply:
Its done. thanks @dasileker